azure_pipelines/verify_msalcpp_per_pr_ios.yml

```diff
@@ -16,7 +16,7 @@ resources:
  repositories:
    - repository: microsoft-authentication-library-for-cpp
      type: github
-     endpoint: 'GitHub for AzureAD and Azure-Samples (as aadidgit service)'
+     endpoint: 'github.com_kaisong1990'
```

Comment:
Is it a correct change? Why it contains "_kaisong1990"? :)

Reply:
Updated with a more general name